src/components/button/Button.tsx

```diff
@@ -54,6 +54,9 @@ const AminoButton = styled.button<ButtonProps<GroupTag>>`
   &:active,
   &:focus {
     outline: none;
+    svg path:not([data-is-secondary-color]) {
+      fill: currentColor;
+    }
```

Comment:
Is all of this "data-is-secondary-color" code directly related to Amino or is this something you specifically used to solve another issue? It's pretty cool!

Reply:
Yeah, it was nice to get that fixed! The "data-is-secondary-color" is related just to the icons in Amino. We did that to solve an issue where the secondary color was being inherited by the parent element and wouldn't display the correct color.